automation/build.sbt

```diff
@@ -43,37 +42,39 @@ Test / testForkedParallel := true
 Test / baseDirectory := (baseDirectory in ThisBuild).value
 
 /*
+ * Each test class is mapped to its own task.
  * Enables (true) or disables (false) parallel execution of tasks.
- * In not-forked mode: test classes are run in parallel in different threads, in same sbt jvm.
- * In forked mode: each test class runs tests in sequential order, in a separated jvm.
  */
-Test / parallelExecution := true
+Test / run / parallelExecution := true
```

Comment:
what does this change do?

Reply:
I'll remove `run`. I was testing something but forgot to remove it.

Reply:
ok